internal/service/organization/data_source_organization_test.go

```diff
@@ -6,21 +6,26 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
 func TestAccConfigDSOrganization_basic(t *testing.T) {
 	var (
-		orgID = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		datasourceName = "data.mongodbatlas_organization.test"
 	)
 	resource.ParallelTest(t, resource.TestCase{
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccMongoDBAtlasOrganizationConfigWithDS(orgID),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttrSet("data.mongodbatlas_organization.test", "name"),
-					resource.TestCheckResourceAttrSet("data.mongodbatlas_organization.test", "id"),
+					resource.TestCheckResourceAttrSet(datasourceName, "name"),
+					resource.TestCheckResourceAttrSet(datasourceName, "id"),
+					resource.TestCheckResourceAttrSet(datasourceName, "restrict_employee_access"),
```

Comment:
shouldn't we also check for the expected value?

Reply:
updated  for settings, can't check for other attributes as ORG_ID may vary